static/src/stylesheets/module/identity/_wizard.scss

```diff
@@ -0,0 +1,27 @@
+.manage-account-wizard {
+    @include clearfix();
+    &--completed {
+        .manage-account-wizard__controls {
+            display: none;
+        }
+    }
+}
+
+.manage-account-wizard__controls {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.manage-account-wizard__controls-pager {
+    @include fs-textSans(1);
+}
+
+@include mq($until: tablet) {
```

Comment:
I think it would be better to inline the mediaqueries in the selectors, so they are not forgotten, when the selectors are removed at some point ...

Reply:
@gustavpursche thanks for the review! i'm on a different branch atm but i'll ping you once i get all of these adressed